scripts/load_static.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export PGPASSWORD=$SCOPULI_PROD_PG_PASSWORD
```

Comment:
```suggestion
export PGPASSWORD=$SCOPULI_PG_PASSWORD
```

After this to that a more generic variable name... you can then call

```
make load_census_divisions.sh SCOPULI_PG_PASSWORD=$SCOPULI_LOCAL_PG_PASSWORD
```